tests/api_test/api_test.cpp

```diff
@@ -56,6 +56,7 @@ _program_load_helper(
     ebpf_result_t result;
     const char* log_buffer = nullptr;
     result = ebpf_program_load(file_name, program_type, nullptr, execution_type, object, program_fd, &log_buffer);
+    printf("Logbuffer: %s\n", log_buffer);
```

Comment:
Why a printf?  If there is expected output, the test should compare log_buffer against the expected output

Reply:
I found it easy to debug when I was struggling to get my test ebpf program loaded. I have changed the test code to print the logbuffer only when ebpf_result != success. I know what the error messages are for the negative test cases. I could put them as REQUIRE on a separate PR. But if verifier implementation changes the error message or if we change the sample programs, then the REQUIRE assertions will fail. 

Let me know if you want the printf statement removed.

Reply:
Removing printf. Adding REQUIRE could be problematic for reasons mentioned above. We could always use netsh ebpf show verification to see load errors.

Reply:
Sorry I really don't understand the problem with REQUIRE.   We already have REQUIRE assertions on error messages in the netsh test.